rest/route/agent.go

```diff
@@ -463,6 +463,7 @@ func (h *getExpansionsAndVarsHandler) Run(ctx context.Context) gimlet.Responder
 		Parameters:  map[string]string{},
 		Vars:        map[string]string{},
 		PrivateVars: map[string]bool{},
+		RedactKeys:  h.settings.LoggerConfig.RedactKeys,
```

Comment:
We will have to bounce the app servers if we update this list since this is using the cached settings. I think that is fine, but just wanted to point it out.